tests/unitary/src/compute/allocator/test_cuda_memory_block.cpp

```diff
@@ -38,9 +38,11 @@ TEST_CASE( "construct cuda_memory_block", "[cuda_memory_block]" )
 {
     const std::uintptr_t ptr_value = 0xDEADBEEF;
     auto *const ptr = reinterpret_cast<void*>(ptr_value);
+    const std::uintptr_t queue_value = 0xA7EBADF0D;
```

Comment:
Where does this value come from?

Reply:
Arbitrary. I usually pick hexpeak values for pointer addresses.

Reply:
But how does that work? I mean, are the hex addresses deterministic? Do you always get the same address regardless of the machine & execution?

Reply:
The address is a constant. I pick readable values (0xDEADBEEF=dead beef 0xA7EBADF0D=ate bad food) for easier traceability in debugging. Accessing these addresses is generally illegal, but here (and other places) they are not accessed, they are used as unique identifiers.